Index: src/pkcs11/Makefile.am
===================================================================
--- src/pkcs11/Makefile.am	(revisión: 412)
+++ src/pkcs11/Makefile.am	(copia de trabajo)
@@ -20,7 +20,7 @@
 
 opensc_pkcs11_la_SOURCES = $(OPENSC_PKCS11_SRC) $(OPENSC_PKCS11_INC) hack-disabled.c
 opensc_pkcs11_la_LIBADD = $(OPENSC_PKCS11_LIBS)
-opensc_pkcs11_la_LDFLAGS = $(AM_LDFLAGS) \
+opensc_pkcs11_la_LDFLAGS = $(AM_LDFLAGS) $(LTLIB_LIBS) \
 	-export-symbols "$(srcdir)/opensc-pkcs11.exports" \
 	-module -shared -avoid-version -no-undefined
 
