Index: src/tools/Makefile.am
===================================================================
--- src/tools/Makefile.am	(revisión: 412)
+++ src/tools/Makefile.am	(copia de trabajo)
@@ -15,7 +15,8 @@
 AM_CFLAGS = $(OPTIONAL_OPENSSL_CFLAGS) $(OPTIONAL_READLINE_CFLAGS) $(PTHREAD_CFLAGS)
 INCLUDES = -I$(top_srcdir)/src
 LIBS = $(top_builddir)/src/common/libcompat.la \
-	$(top_builddir)/src/libopensc/libopensc.la
+	$(top_builddir)/src/libopensc/libopensc.la \
+        $(top_builddir)/src/common/libltdl.la
 
 opensc_tool_SOURCES = opensc-tool.c util.c
 piv_tool_SOURCES = piv-tool.c util.c
